### DISCUSSION TITLE
Please add this to the docs on how to connect with valkey-cli/redis-cli from wsl2, to a garnet-server running on windows

### DISCUSSION THREAD

**Discussion by jmwielandt** [microsoft/garnet#844] [Show and tell]
Hi

There is a crucial detail missing [here](https://microsoft.github.io/garnet/docs/getting-started).
![image](https://github.com/user-attachments/assets/0a70f44a-21e3-445d-9765-2571f4c08672)

If you are using wsl2 on windows, you may need to:
1. cd to ~
2. `new-item .wslconfig` (if it doesn't exist)
3. open the .wslconfig
4. add the following:
   ```ini
   [wsl2]
   networkingMode=mirrored
   ```
5. save the file
6. wsl --shutdown

Now, you can go to the wsl2 and run `valkey-cli` and the connection will work.

My environment:
![image](https://github.com/user-attachments/assets/5133c73b-1137-451d-8326-3ad90f0c7cbb)
![image](https://github.com/user-attachments/assets/8ace64bc-e1ae-417a-8015-5600dd2d4867)

**Comment by badrishc** [comment]
Mirrored networking mode works only on Windows 11. Instead, here is how we make WSL clients work with a Garnet Windows server in general:

* Start GarnetServer in Windows, listening to `0.0.0.0` (`IPAddress.Any`). Use the argument `--bind 0.0.0.0` if necessary.
* Add `GarnetServer.exe` to firewall exceptions (Windows Defender Firewall -> Allowed apps)
* On WSL, get the host IP address using `ip route show | grep -i default | awk '{ print $3}'`
* Connect from WSL using `redis-cli -h <address>`
